README.md

```diff
@@ -174,8 +203,107 @@ When geolocating, if both targeting keys and geolocation columns are set, we wil
 - IF only targeting columns are set, AND targeting conditions are fulfilled
   - `sorcerer` will use targeting
 
+### Multiple target retrieval for CSV Data Sources
+
+To fetch multiple targets from a CSV DataSource you can use the `getMultipleTargets` method, which will return you an array of objects based on the number of rows that match. 
+
+#### Example
+
+If our CSV file is like below:
+
+Level | Tier | Content
+-- | -- | --
+1 | Silver | Marry Poppins
+2 | Gold | Peter Pan
+3 | Platinum | Robin Hood
+1 | Silver | Tom and Jerry
+5 | Gold | Aladdin
+1 | Gold | The Lion King
+1 |   | Cinderella
+
+You can make a request like this: 
+
+```js
+const options = {
+  method: 'POST', // method has to be POST
+  body: JSON.stringify([
+    {
+      Level: 1,
+      Tier: 'Silver',
+    },
+    {
+      Level: 2,
+      Tier: 'Gold',
+    },
+  ]),
+};
+
+const source = new DataSource('some_key');
+const data = await source.getMultipleRows(options);
```

Comment:
looks like you named the actual method `getMultipleTargets`?

Reply:
Ahhh yes, bad copy & pasta